Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -237,20 +258,21 @@
                     .withTitle("Status ifølge datofelter")
                     .withId("isActive")
                     .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
-                    .withFixedValueRange([
-                        {
-                            textValue: "Aktivt",
-                            remoteValue: true
-                        },
-                        {
-                            textValue: "Ikke aktivt",
-                            remoteValue: false
-                        }
-                    ],
-                        false)
-                    .withRendering(dataItem => dataItem.ActiveAccordingToValidityPeriod ? 'Aktivt' : 'Ikke aktivt')
+                    .withFixedValueRange(createActiveRange(), false)
+                    .withRendering(dataItem => dataItem.ActiveAccordingToValidityPeriod ? active : notActive)
```

Comment:
looks like we could use a `renderActiveNotActive(value: boolean)`
That can also be reused between the "so far" two columns